scipy/stats/tests/test_continuous_basic.py

```diff
@@ -401,18 +401,18 @@ def test_nomodify_gh9900_regression():
     # Use the right-half truncated normal
     # Check that the cdf and _cdf return the same result.
     npt.assert_almost_equal(tn.cdf(1, 0, np.inf), 0.6826894921370859)
-    npt.assert_almost_equal(tn._cdf(1, 0, np.inf), 0.6826894921370859)
+    npt.assert_almost_equal(tn._cdf([1], [0], [np.inf]), 0.6826894921370859)
```

Comment:
All `rv_continuous` private methods should be able to assume they are getting arrays that are at least 1d because that's what they are passed from the public method. The implementation of `_logcdf` relies on this, so rather than add an additional `np.atleast_1d` call in `_logcdf`, ensure the input is at least 1d here.